pkgs/os-specific/linux/lenovo-legion/app.nix

```
@@ -1,37 +1,45 @@
-{ lib, fetchFromGitHub, xorg, libsForQt5, wrapQtAppsHook, python3 }:
+{
+  lib,
+  fetchFromGitHub,
+  xorg,
+  wrapQtAppsHook,
+  python3,
+}:
 
 python3.pkgs.buildPythonApplication rec {
   pname = "lenovo-legion-app";
-  version = "0.0.12";
+  version = "0.0.18";
```

Following the [versioning convention](https://github.com/NixOS/nixpkgs/blob/0a0c9e56cd7cfe1b18745fd5c6c741a3d1b6a0ca/pkgs/README.md#versioning):

```suggestion
  version = "0.0.18-unstable-2024-09-22";
```

The commit title should also be changed to:
```
lenovo-legion: 0.0.12 -> 0.0.18-unstable-2024-09-22
```

---

Nevermind, this apparently isn't working...

```
  File "/nix/store/l7idy2qiiv0v0b6khfjvz3l5k6mnm47l-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/_vendor/packaging/version.py", line 202, in __init__
    raise InvalidVersion(f"Invalid version: '{version}'")
packaging.version.InvalidVersion: Invalid version: '0.0.18-unstable-2024-09-22'
```

---

> This should have been set but in postPatch we should only set 0.0.18

I have created a separate pr for this https://github.com/NixOS/nixpkgs/pull/349682